### PR TITLE
[Sql Server] (No.5) add open in portal and delete actions

### PR DESCRIPTION
**Pull request by shenqianjin**

### CODE REVIEW

File: Utils/azure-explorer-common/src/com/microsoft/tooling/msservices/serviceexplorer/azure/sqlserver/SqlServerModule.java [hunk 1]
```diff
@@ -46,9 +45,7 @@ protected void refreshItems() throws AzureCmdException {
 
     @Override
     public void removeNode(String sid, String id, Node node) {
-        /**
-         * TODO(qianjin): implementation remove logic.
-         */
+        Azure.az(AzureSqlServer.class).sqlServer(id).delete().commit();
```
Comment (andxu) [inline Utils/azure-explorer-common/src/com/microsoft/tooling/msservices/serviceexplorer/azure/sqlserver/SqlServerModule.java]: delete();

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/lib/sqlserver/SqlServerConfig.java [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    private Region region;
+
+    private String adminUsername;
+    private char[] password;
+    private char[] confirmPassword;
+
+    private boolean allowAccessFromAzureServices;
+    private boolean allowAccessFromLocalMachine;
+
+    public static SqlServerConfig getDefaultConfig() {
```
Comment (andxu) [inline PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/lib/sqlserver/SqlServerConfig.java:42]: why create the default config here(POJO)

Comment (shenqianjin) [reply PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/lib/sqlserver/SqlServerConfig.java:42]: initialize default values for sql server creation.

File: Utils/azure-explorer-common/src/com/microsoft/tooling/msservices/serviceexplorer/azure/sqlserver/SqlServerModule.java (hunk 1 above)
Comment (shenqianjin) [reply Utils/azure-explorer-common/src/com/microsoft/tooling/msservices/serviceexplorer/azure/sqlserver/SqlServerModule.java]: updated.

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/lib/sqlserver/SqlServerConfig.java [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    private char[] confirmPassword;
+
+    private boolean allowAccessFromAzureServices;
+    private boolean allowAccessFromLocalMachine;
+
+    public static SqlServerConfig getDefaultConfig(Subscription subscription) {
+        final String defaultNameSuffix = DateFormatUtils.format(new Date(), "yyyyMMddHHmmss");
+        final SqlServerConfig config = new SqlServerConfig();
+        config.setSubscription(subscription);
+        DraftResourceGroup resourceGroup = DraftResourceGroup.builder().subscription(subscription).name("rs-" + defaultNameSuffix).build();
```
Comment (andxu) [inline PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/lib/sqlserver/SqlServerConfig.java:46]: DraftResourceGroup -> ResourceGroup

**Review by andxu — APPROVED** [review]

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/lib/sqlserver/SqlServerConfig.java (hunk 3 above)
Comment (shenqianjin) [reply PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/lib/sqlserver/SqlServerConfig.java:46]: please merge resourcegroup/region/subs lib as soon as you can. I will refactor the code according by it.

**Pull request merged**